src/components/App/App.jsx

```diff
@@ -26,16 +25,22 @@ class App extends React.Component {
     ipcRenderer.send('title', 'Notifications')
     this.setupAppMenu()
     if (GitHubAuth.isAuthenticated()) {
+      this.loadUser()
       const key = LastFilter.retrieve()
       if (key) {
         this.loadFilter(key)
       } else {
         this.manageFilters()
       }
-      this.loadUser()
     }
   }
 
+  onUserLoad(user) {
```

Comment:
Maybe rename to `login` so it's clear you're not getting a full user object, just a string?